liquibase-core/src/main/java/liquibase/change/core/DropForeignKeyConstraintChange.java

```diff
@@ -10,14 +10,15 @@
 /**
  * Drops an existing foreign key constraint.
  */
-@DatabaseChange(name="dropForeignKeyConstraint", description = "Drops an existing foreign key", priority = ChangeMetaData.PRIORITY_DEFAULT, appliesTo = "foreignKey")
+@DatabaseChange(name = "dropForeignKeyConstraint", description = "Drops an existing foreign key", priority = ChangeMetaData.PRIORITY_DEFAULT, appliesTo = "foreignKey")
```

Comment:
The word "existing" is not required; it's implied that the FK exists if you are dropping it.

Reply:
Agreed, though if we make this change we should be consistent with descriptions on `dropTableChange` etc. ("Drops an existing table" -> "Drops a table.") Sometimes it's useful to use emphasis words like "new" or "existing" but I try to only do this as a point of comparison (new/old, existent/not-yet-existent, etc) or if a description could otherwise be confusing without the terms.